### ISSUE TITLE
CS224N_Week1 조민영 질문

### ISSUE THREAD

**Issue by chominyeong** [ckj18/NLP2Study#6]
강의에서 RNN(Recurrent Neural Network)은 시간 t일 때 RNN의 hidden state인 h_t를 구하기 위해 input data xt에 가중치 Wx를 곱하고, 이전 hidden state인 h_(t-1)에 가중치 Wh를 곱하여 둘을 더한 값을 어떤 함수에 넣어 구할 수 있다고 했다. h(t) = δ(Wh*h_(t-1) + Wx*xt + b1)

어떤 함수가 선형(δ(x)=x)일 BPTT(Backpropagation Through Time) 알고리즘을 적용하면 에러가 발생한 시점에서 멀리 떨어진 과거 시점일수록 에러의 그래디언트가 전달되지 않는 Vanishing gradient 문제를 가지게 된다. (p54)

만약 선형함수 대신 비선형함수를 사용한다면 vanishing gradient 문제를 예방할 수 있는지 궁금하다.
